src/app/page.tsx

```diff
@@ -1,12 +1,28 @@
-import BottomNav from '@/components/bottom-nav/BottomNav';
+'use client';
 import styles from './page.module.css';
+import ActiveButton from '@/components/button/ActiveButton';
+import InactiveButton from '@/components/button/InactiveButton';
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <h1>Welcome to Deer Tracker!</h1>
-      <h3>Powered by AI & Machine Learning</h3>
-      <BottomNav />
+    <main className={styles.mainpageContainer}>
+      <div className={styles.headerconatiner}>
+        <h1 className={styles.welcomecontainer}>Welcome to Deer Tracker!</h1>
+      </div>
+
+      <div className={styles.lgnbttncontainer}>
+        <ActiveButton
+          text="Log in"
```

Comment:
define this string as a constant variable in constant.ts and then import the variable into this file